charts/oes/charts/spinnaker/templates/statefulsets/halyard.yaml

```diff
@@ -41,7 +41,7 @@ spec:
       initContainers:
       - name: "create-halyard-local"
       {{- if .Values.gitopsHalyard.enabled }}
-        image: alpine/git:v2.26.2
+        image: quay.io/opsmxpublic/awsgit
```

Comment:
Let's version this image and use corresponding tag here

Reply:
done